training/elf/elf_convert.py

```diff
@@ -41,7 +41,10 @@ def write_block(f, b):
 with open('elf_converted_weights.txt', 'w') as f:
     # version 2 means value head is for black, not for side to move
     f.write('2\n')
-    b = convert_block(state, 'init_conv')
+    if 'init_conv.0.weight' in state:
```

Comment:
You need to rebase.

Reply:
Evidently. Sorry, I'm rather a git novice. I've pushed a change that addresses the other review comments; I'll fix this one once I'm confident of not breaking everything when I try :-).

Reply:
I _think_ I've fixed this now. But, again, git novice, may have broken everything without noticing.